### ISSUE TITLE
Iguana Tweaks leveling

### ISSUE THREAD

**Issue by Grovert11** [MikeLydeamore/Ex-Astris#80]
I don't know for sure of this is on your side. When using a tool with the Smashing modifier, the Iguana Tweaks xp level won't go up.

**Comment by LoveHoly** [comment]
Iguana Tweaks compatibility?

**Comment by Grovert11** [comment]
Yup, that souns like it! :D Yesterday I discovered that it dóes work when shoveling/hammering gravel and sand, so it's only the ore gravels (which I made harvestable with a shovel myself with IguanaTInkerTweaks).
